socioeconomic-differences/socioeconomic-differences.md

```diff
@@ -12,66 +12,88 @@ A person's socioeconomic status can inform their identity, as it affects the com
 
 
 ### Independent Practice
+    
+	1.The students can further read articles,books,listen to podcasts or watch videos to enhance their understanding of the subject.
+    
+### Challenge (2 hrs research after the class,preferably done in group)
+     
+	1.Students can take a survey in their neighborhood(they can work in the group of 4), working places etc regarding people who identify themselves as "Class Straddlers" or people who have started working white-collar jobs after their previous blue-collar job and ask questions such as:-
+	a.  What is the percentage of increase in their salary after the shift in the job?
+	b.  What motivated them to change their profession?
+	c.  What are the major differences they feel with the transition of profession or lifestyle?
+  d.  Are they satisfied with the experience in their new work space?
+	e.  What are the set of new skills that they had to acquire to get the white-collar job?
+	f.  How did they learn the above mentioned skills?
+	(Further questions can be added as suitable)
+	The following data can be accumulated,with common reason binded together and thus the following data can be used to gain great deal of knowledge about the main issues,challenges and progress in their respective society.
+    The following collected data can also be used for further assignment or data-science project.
+
+	2.Students can work together to collect data to find which state in their country saw a major conversion of people from blue-collar job to white-collar job and what were the factors that was responsible, for e.g policies of their local government, increase of jobs due to sudden growth of offices in the areas, reason the offices were setup.  
+   
+### Check for Understanding(37-40 minutes)
+
+    1. A discussion can be held in the end where the participants summarize each other.
+    2. Ask the students to read the first 30 pages of "Blue-Collar Roots, White-Collar Dreams" by Alfred Lubrano,and narrate the essence of what they have read and understood so far.  
+    3. Ask the students  specifically regarding the problems one faces in order to achieve promotion as "Class Straddlers".Ask them what they think are the appropriate measures to tackle the mentioned problems.
+    4. Give assignment to the students to read atleast 2 short stories(try to assign the following assignment in group) from " Hillbilly Elegy " by "J.D. Vance" and ask them to explain the problems regarding "work ethics" as narrated in many of  the stories mentioned in the book and ask them to come up with solutions to tackle such issues. 
+
+### Supplemental Materials(15 minutes)
+	  1. <https://genprogress.org/straddling-college-and-working-class/> - The following article derives a narrative about the problems working-class students face in elite schools and furthermore the effect of the problem may hinder their jobs too.
```

Comment:
please use markdown for these links.

Reply:
still waiting for this fix.

Reply:
supplemental materials is optional, so please remove the time estimate. Feel free to add time estimates to each individual link however; it's good to know what to expect.